Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -402,6 +402,8 @@ std::optional<Protocol::CSS::PseudoId> InspectorCSSAgent::protocolValueForPseudo
         return Protocol::CSS::PseudoId::FirstLine;
     case PseudoId::FirstLetter:
         return Protocol::CSS::PseudoId::FirstLetter;
+    case PseudoId::GrammarError:
```

Comment:
youll also need to add these to `WI.CSSManager.displayNameForPseudoId` if you want them to actually appear in Web Inspector

Reply:
Thanks!